server/app/lib/actions/fusor/deployment/open_shift/install_image.rb

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+require 'net/scp'
+
+module Actions
+  module Fusor
+    module Deployment
+      module OpenShift
+        class InstallImage < Actions::Fusor::FusorBaseAction
+          def humanized_name
```

Comment:
Agree, with the name change, transfer_image.rb makes more sense than install_image.rb  Originally this task was going to leverage a yum install to get the content, but it since changed.